tests/api/test_security.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import unittest
+import os
+
+
+class Testsecurity(object):
+    def setup(self):
+        # app['TESTING'] = True
+        # self.app = app.test_client()
+        pass
+
+    def test_main_page(self, flask_api_url):
+        response = requests.get(flask_api_url + '/')
+        assert response.status_code == 200
+
+    def test_login_fake_user(self, flask_api_url):
+        response = requests.post(flask_api_url + '/api/users/login',
```

Comment:
uh, you could just try deleting the user right before the call, that way it's guaranteed to not be in the database. however, since the tests are mainly for ravis I don't really see why we would need to, it's a clean build with only the default users (user&admin@knex) anyway